src/icons/imported/Icon/IconBoxApp512.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+/* eslint-disable react/jsx-sort-props */
+import * as React from 'react';
+import AccessibleSVG from '../../accessible-svg';
+
+type SvgProps = {
+    height?: number,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconBoxApp512 = (props: SvgProps) => (
+    <AccessibleSVG width={32} height={32} viewBox="0 0 512 512" {...props}>
```

Comment:
The component file name has `512` in its name, though it seems like it's 32x32px. Do we need the dimensions in the file name?

Reply:
From design's perspective, the number in the name is just to mark the resolution, not the default size it should use. The import script is currently naïve in this regard (it always sets the default to 32x32), tho it does have the ability to account for per-asset details (and viewbox metrics could be part of that).

Thoughts?

Reply:
went ahead and added ability to read/use the viewBox values for the default prop values to the importer